content/github/administering-a-repository/renaming-a-branch.md

```diff
@@ -38,4 +38,10 @@ From the local clone of the repository on a computer, run the following commands
 $ git branch -m <em>OLD-BRANCH-NAME</em> <em>NEW-BRANCH-NAME</em>
 $ git fetch origin
 $ git branch -u origin/<em>NEW-BRANCH-NAME</em> <em>NEW-BRANCH-NAME</em>
+$ git remote set-head origin -a
+```
+
+Run the following command to remove tracking references to `OLD-BRANCH-NAME`:
```

Comment:
I'm not sure if I should use \` \` formatting for "OLD-BRANCH-NAME", or italicized text (or italicized _and_ code formatting)... So this is what I went with -- feedback is obviously welcome 😄

Reply:
👋🏼  @alisonjo315, this is a tricky case since `OLD-BRANCH-NAME` is both a branch name and dynamic text. You've written it exactly right since branch names use code formatting ` ` and dynamic text is capitalized. No need for italics here. ✨ 

Our docs can be inconsistent with how some of these elements are written--I know it can cause confusion internally on the team 😅 --so it's super helpful to have a contribution that follows the style guide and will help make the docs more consistent overall! Thank you 🎉 

Style guide references:
[Branch names](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#branch-names)
[Dynamic text](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#dynamic-text)